docs/source/agent.rst

```diff
@@ -0,0 +1,63 @@
+Elastic Agent
+==============
+
+.. automodule:: torchelastic.agent
+.. currentmodule:: torchelastic.agent
+
+Server
+--------
+
+.. automodule:: torchelastic.agent.server
+
+Below is a diagram of an agent that manages a local group of workers.
+
+.. image:: agent_diagram.jpg
```

Comment:
Hmm that is strange that it's not rendering because this syntax looks correct. Have you you tried these 2 options just to test if it picks up?

1. Try writing out either the full path:
``.. image:: /docs/source/agent_diagrams.jpg``

2. Move the image over to another directory and point it there:
``... image:: ../_static/img/agent_diagram.jpg``